pyvo/dal/tests/test_tap.py

```diff
@@ -477,14 +478,16 @@ def test_get_job_list(self):
         # Tests consists in counting the cumulative number of jobs as per
         # above rules
         after = datetime.datetime.now()
-        assert len(service.get_job_list()) == 0
-        assert len(service.get_job_list(last=3)) == 3
-        assert len(service.get_job_list(after='2018-04-25T17:46:01Z')) == 1
-        assert len(service.get_job_list(phases=['EXECUTING'])) == 2
-        assert len(service.get_job_list(after=after,
-                                        phases=['EXECUTING'])) == 3
-        assert len(service.get_job_list(after='2018-04-25T17:46:01.123Z',
-                                        last=3)) == 4
-        assert len(service.get_job_list(phases=['EXECUTING'], last=3)) == 5
-        assert len(service.get_job_list(phases=['EXECUTING'], last=3,
-                                        after=datetime.datetime.now())) == 6
+
+        with pytest.warns(UnknownElementWarning):
```

Comment:
This should not happen. I'm currently working to find a fix and might remove it in the future.

Reply:
thanks for the fix. I'll remove this workaround and rebase this PR once that other one is merged.

Reply:
> @andamian - The one big item missing in this PR is to add a `remote-data` CI job (#330). That would be a big PR as it will need to fix all doctests as right now most of them are skipped as the examples are reaching out to remote services.

100% agreed

Reply:
You don't have to do that. If this is ready we can merge it first and I'll deal with this in the other PR. That's why I said it's a race... :-)

Reply:
This is my last comment. I could merge it afterwards unless you want Tom's feedback as well.